bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java

```diff
@@ -1014,9 +1032,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
         }
 
         if (cause instanceof IOException) {
-            // these are thrown when a bookie fails, logging them just pollutes
```

Comment:
Any strong reason for dropping the comment and adding this log?

Reply:
I introduced a WARN log, so removed the comment. I expect exceptions should be a rare event so we should not see them often in the logs.